docs/pages/upgrade-to-v3.md

```diff
@@ -3,6 +3,7 @@
 The process to upgrade an app built with XMTP V2 to V3 is designed to be straightforward, with most functions in V3 working as they did in V2. However, there are some notable differences, which we cover here.
 
 :::info[Key takeaways]
+- **Primary XMTP identifier is now a flexible identity object, not an Ethereum address** as covered in this document.
```

Comment:
Primary XMTP identifier is now a inboxId instead of an Ethereum address. This inbox can have a list of identities including ethereum addresses as well as other types in the future like passkeys and bitcoin.